### DISCUSSION TITLE
Re-deploying into a private VPC  and private network.

### DISCUSSION THREAD

**Discussion by scobatch** [aws-solutions/workload-discovery-on-aws#518] [Q&A] [answered]
Hey there,  I had to tear down the Discoverer stack and redeploy in the private vpc and private network.  But now when I attempt to redeploy within the private vpc I am running into the below issue. I actually opened a support ticket with AWS for this and they seem to be at a loss as to what is causing this issue.   Just curious if anyone else has ran into this?  Any help would be greatly appreciated!

Embedded stack arn:aws:cloudformation:us-east-1::stack/workload-discoverers-SearchResolversStack-1U5089BFP2IS7/dfd2a020-f1c8-11ee-bc4b-0e3a215ac417 was not successfully created: Stack creation time exceeded the specified timeout

**Comment by scobatch** [comment, marked as answer]
I fixed this.  For some reason a route table was not added to the S3 Gateway Endpoint during the last deployment.

**Comment by svozza** [comment]
Apologies for the late reply, I was on PTO. Yeah, the S3 gateway endpoints work slightly differently to interface endpoints in that they need route table information: I think we should add that info to the [VPC pre-requisite steps](https://docs.aws.amazon.com/solutions/latest/workload-discovery-on-aws/prerequisites.html#verify-your-aws-config-details-in-your-account) of the WD documentation. Another point, when running in a VPC with no internet access is that the discovery process needs access to quite a few AWS service APIs. There is a [full list in the documentation](https://docs.aws.amazon.com/solutions/latest/workload-discovery-on-aws/aws-apis.html) so you will need to add a VPC endpoint for each service listed.

**Comment by aws-ymson** [comment]
Closing this out: Workload Discovery on AWS has been deprecated as of 08/14/2026 and is no longer being maintained.
